code/modules/clothing/clothing.dm

```diff
@@ -217,6 +217,23 @@
 		for(var/A in armor)
 			armor[A] -= rand(armor[A]/3, armor[A])
 
+/obj/item/clothing/attack(mob/living/M, mob/living/user, def_zone, var/originator = null)
+	if(iscarbon(user))
```

Comment:
This is FINE as it is, but I personally prefer returning early in case the conditions aren't what we expect, rather than nesting if() under if() under if(). This would look something like 
```
if(!iscarbon(user))
   return
```
And so on.
Purely a code style nitpick, feel free not to address this.

Reply:
I don't really understand what you're getting. If we do
	`if(!iscarbon(user))
		..()
	else`
What happens we are indeed a carbon but don't want to help ?